source/administration-guide/configure/calls-deployment.md

```diff
@@ -127,6 +127,26 @@ Mattermost Calls can function in air-gapped environments. Exposing Calls to the
 - Configuring a STUN server is unnecessary, as all connections occur within the local network.
 - The [ICE Host Override](https://docs.mattermost.com/configure/plugins-configuration-settings.html#ice-host-override) configuration setting can be optionally set with a local IP address (e.g., 192.168.1.45), depending on the specific network configuration and topology.
 
+### RHEL deployments with firewalld and fapolicyd
+
+When deploying Mattermost Calls or the rtcd service on RHEL systems with firewalld or fapolicyd enabled, additional firewall and security configuration is required.
+
+**Required ports:**
+- **Calls plugin (integrated)**: Port 8443 UDP/TCP for RTC media traffic
+- **rtcd service (standalone)**: Ports 8443 UDP/TCP for RTC media traffic + Port 8045 TCP for API communication with Mattermost
+
+**Configuration steps:**
+
+For complete firewalld and fapolicyd configuration instructions, including troubleshooting steps and example rules, see the [RHEL deployment guide](https://docs.mattermost.com/deploy/server/deploy-rhel.html). The guide includes:
```

Comment:
Let's link to the current home of the RHEL deployment guide on GitHub. We can update this link once that content moves over to the Product Docs site.

Reply:
OK. In the generated preview, that URL isn't resolving as expected. This will work: https://docs.mattermost.com/deployment-guide/server/deploy-linux.html#itab--RHEL-CentOS--0_1-RHEL-CentOS